test/test_ast_generation.py

```diff
@@ -523,6 +523,12 @@ def f(*a, b):
     ("f-string_lambda", "f'{lambda x: 42}'"),
     ("f-string_singe_brace", "f'{'"),
     ("f-string_single_closing_brace", "f'}'"),
+    ("invalid indentation",
```

Comment:
(Maybe add a comment that this test is really testing the uninitialized col_offset issue this bug is fixing?)